solutions/olof/stone.js

```diff
@@ -0,0 +1,114 @@
+let valElements = document.querySelectorAll(".val");
+
+const stenp1 = document.getElementById("stenp1");
+const saxp1 = document.getElementById("saxp1");
+const påsep1 = document.getElementById("påsep1");
+
+const stendata = document.getElementById("stenData");
+const saxdata = document.getElementById("saxData");
+const påsedata = document.getElementById("påseData");
+
+const playAgainButton = document.querySelector("#playAgainButton");
+const getLostElements = document.querySelectorAll(".getLost");
+
+let p1Val = undefined;
+let dataVal = undefined;
+
+
+stenp1.addEventListener('click', function () {
+    p1Val = "sten";
+    stenp1.style.backgroundColor = "pink";
+    setTimeout(() => {
+        dataValAndCheckResult();
+    }, 1432);
+});
+
+saxp1.addEventListener('click', function () {
+    p1Val = "sax";
+    saxp1.style.backgroundColor = "pink";
+    setTimeout(() => {
+        dataValAndCheckResult();
+    }, 1432);
+});
+
+påsep1.addEventListener('click', function () {
+    p1Val = "påse";
+    påsep1.style.backgroundColor = "pink";
+    setTimeout(() => {
+        dataValAndCheckResult();
+    }, 1432);
+});
+
+
+let datapoäng = 0;
+let p1poäng = 0;
+
+const dataValAndCheckResult = function () {
+    dataVal = Math.floor(Math.random() * 3);
```

Comment:
`3` could be exchanged for some arrays length. This make it more flexible to change.